timereport/app.py

```diff
@@ -124,6 +124,27 @@ def index():
         return ''
 
     if action == "list":
+        app.log.debug(f"Running list action with: {params}")
+        if 'total' in params:
+            app.log.debug('get total workdays here for year/month')
+            # params should be ['total', '2019', '02']
+            try:
+                validate_year(params[1])
+                validate_month(params[2])
+                year = params[1]
+                month = params[2]
+            except:
```

Comment:
Kind of dangerous to have this broad catch of exception.
Better to catch the known exception and let the others crash the "app".